### PR TITLE
Manolo tlc ciel d8 3b

### PR DESCRIPTION
**Pull request by nolo16**
this took me fucking forever for some reason

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 1]
```diff
@@ -217,17 +218,17 @@ Incidentally, Senpai unexpectedly orders a simple chai tea.
 -- Page 265, Offset 18930.
 -- ここまできたんだから、安定のいつものネタをやってくれないと。
 // Not sure if it's clear enough but "安定のいつものネタをやってくれないと" means that Ciel has to say usual comedy routine line (which is of course that she's going to eat curry)
-Now that I've said that much, she has to follow up with her usual line.
+Now that I've said this much, she's got to deliver the usual punchline.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:221]: this is good. delete the comment if you would

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 2]
```diff
@@ -237,17 +238,18 @@ The butter chicken was finger-licking good.
 [sha:5ba48769cc2c8e32b2e66e9dfddc959ec5a7908f]{
 -- Page 267, Offset 18934.
 -- カレーソースはバターが効いているのか辛いのに甘いという絶妙なバランスで、ナンも熱く柔らかく、パン生地を噛みしめるだけで旨みが口内に溢れてくる。
-The curry sauce was an exquisite balance of spiciness and sweetness, thanks perhaps to the butter. The naan was pipping hot and soft so that just by taking a bite of the bread, I could feel the savoriness spreading all over my taste buds.
+// 旨み to "savoriness" here, dunno if umami is common knowledge enough to keep as is
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:241]: umami is a common expression in english and is distinct from savoriness. just transliterate it

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 3]
```diff
@@ -237,17 +238,18 @@ The butter chicken was finger-licking good.
...
 -- Page 268, Offset 18935.
 -- この店は当たりだ。
-That restaurant is a gem.
+This restaurant is a gem.
 }
 [sha:1fb24a817b52c5ff99c65ab7b3503f2b6febb22e]{
 -- Page 268, Offset 18936.
 -- カレーショップ・メシアン、覚えておこう。有彦にも教えてやろう。
-I'll keep note of you, curry shop 'Messiaen'. I'll recommend it to Arihiko too.
+I'll remember you, curry shop 'Messiaen'. I'll tell Arihiko about it too.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:252]: is this supposed to be curry shop messiah? Or is he referencing the composer? this is a weird line

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 4]
```diff
@@ -309,17 +310,17 @@ It was a lightning-fast movement, almost as if it was of the utmost importance.
...
 -- Page 273, Offset 18949.
 -- 「え―――ご、ごめんなさい、つい釣られちゃって……！」
-"Eh? I-I'm sorry, I was so lured...!"
+"Eh? I-I'm sorry, I was lured in..!"
 }
 [sha:23546edcd305ad726c2e848c1b841aadd0501133]{
 -- Page 273, Offset 18950.
 -- うん、ホントに大漁でした。
-Yeah, she really is a big fish.
+Yeah, I've seriously hauled in a good catch.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:323]: i will be referring to ciel as "big fish" from now on

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 5]
```diff
@@ -329,17 +330,17 @@ Senpai is sitting with her shoulders hunched up from shame.
 [sha:827787c4be2f2a721f29b892713d2c5ea12e58a7]{
 -- Page 273, Offset 18952.
 -- こっちとしてはたいへんいい先輩が見られたので、ぜんぜんごめんなさいではない。
-As for me, I got to see a really nice version of Senpai, so I'm not at all sorry.
+As for me, I got a glimpse of Senpai's excellence, so I don't feel remorseful at all.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:333]: so I don't feel even an ounce of remorse

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 6]
```diff
@@ -509,12 +511,12 @@ That's why I wanted to remember it.
...
 -- Page 288, Offset 18988.
 -- ……俺には、この人ほどの強さはないにしても。
-Even if I don't have anywhere near her strength.
+Even if I'm nowhere near as strong as her.
 }
 [sha:d83686e5fe17a908b3ffe4f5dedff5a0caa4e699]{
 -- Page 288, Offset 18989.
 -- 誰に感謝される事もなかったあの奮起を、せめて俺ひとりぐらいはありがとうと言えるように、ずっと覚えていたいと願ったんだ。
-I wish to forever remember her courage, for which no one had ever been grateful, so that at least someone, at least I can thank her.
+I wish to forever remember her courage. Her courage that has not once been rewarded by anyone, so that at least I alone can thank her.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:519]: god damn is this line jank. Here's an alternate tl which is also jank
> I want to forever remember that courage which has gone unappreciated for so long. That will be my way of saying thank you.

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 7]
```diff
@@ -664,6 +667,7 @@ I can use Black Keys, barriers, sacraments too, to some extent... I think."
 [sha:472f57687fb43dec96d09ace94aa4fe3f3698f80]{
 -- Page 302, Offset 19020.
 -- 「そ、そうなんだ。俺は視る事しかできないけど、先輩はやっぱり色々できるんだね」
+// do we need to explicitly add "lines" here?
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:670]: not sure how else this line would be clear tbh. Maybe
> I-Is that so? All I can do is see things. I guess it's no surprise that you're proficient in so many things, Senpai.

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 8]
```diff
@@ -776,8 +780,8 @@ After all, death is permeating this entire world.
 [sha:2c412bb92c25e6878fc10d55b9aa358feae1d648]{
 -- Page 312, Offset 19042.
 -- ある司祭からの受け売りですが、<無|し>とはこの宇宙にいまだないもの。在ると想定する事で多くの事象を説明できるけれど、実在してはいけないものとされています」
-// Universe? Cosmos?
-I heard this from a priest, but <nothingness|d e a t h   > is something that doesn't yet exist in our universe. Supposing its existence allows us to explain many phenomenons, but it is seen as something that must not exist."
+// this is going in one ear and out the other
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:783]: lmfao what

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 9]
```diff
@@ -787,7 +791,8 @@ I solemnly nod in comprehension.
 [sha:99967a2c9986d40bfa9d98236d25e2b068016ead]{
 -- Page 313, Offset 19044.
 -- そんな俺をじーっと見つめるスパルタな先輩。
-Senpai stares at me gravely thus.
+// dont know if this is right for スパルタ but dont have anything better
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:794]: I don't really know what the hell スパルタな means here. lit. "my Spartan Senpai stares at me fixedly". I have no fuggin idea what that means.

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 10]
```diff
@@ -797,17 +802,18 @@ Senpai stares at me gravely thus.
...
 -- もう少しレベルをさげて教えてください」
-Could you explain once more but dumbed down?"
+Could you explain that again, but dumbed down?"
 }
 [sha:679f2a254e8217669f103ed9abc4f67633a36084]{
 -- Page 315, Offset 19047.
 -- 「つまり空想上のものです。ファンタジーですね。ありきたりのものどころか、どこにもないモノってコトです。
-"In short, it is an object of the imagination. Call it fantasy, not something common, but something that cannot be found anywhere.
+// ???
+"In short, it is something above imagination; a fantasy. Though it is a commonplace concept, it does not exist anywhere.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:811]: imagination: a fantasy

**Review by Avuxo — APPROVED** [review]
well that file was sure somethin, huh.
203 lines of _somethin_

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt (hunk 3 above)
Comment (nolo16) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:252]: i had to do a double take too, but the name is correct

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt (hunk 9 above)
Comment (nolo16) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:794]: I think it means something along the lines of "merciless" here

https://dictionary.goo.ne.jp/word/%E3%82%B9%E3%83%91%E3%83%AB%E3%82%BF/

**Pull request merged**